mkdocs.yml

```diff
@@ -30,6 +30,8 @@ nav:
       - 'Running Renovate': 'getting-started/running.md'
       - 'Installing & Onboarding': 'getting-started/installing-onboarding.md'
       - 'Private Packages': 'getting-started/private-packages.md'
+  - Troubleshooting:
+      - 'Troubleshooting': 'troubleshooting.md'
```

Comment:
```suggestion
  - Troubleshooting: 'troubleshooting.md'
```

I think this is the correct way to place just one item on the sidebar?

Reply:
I think so